### ISSUE TITLE
readme is wrong when talking about storage configuration.

### ISSUE THREAD

**Issue by niemyjski** [elastic/elasticsearch-cloud-azure#78]
I'm using 1.5.1 and the following is wrong:

```
To enable Azure repositories, you have first to set your azure storage settings in elasticsearch.yml file:

cloud:
    azure:
        storage:
            account: your_azure_storage_account
            key: your_azure_storage_key
For information, in previous version of the azure plugin, settings were:

cloud:
    azure:
        storage_account: your_azure_storage_account
        storage_key: your_azure_storage_key
```
The previous version is the only version that works for me on 1.5.1

**Comment by dadoonet** [comment]
Are you talking about 2.5.1?

If so, the right documentation lives here: https://github.com/elastic/elasticsearch-cloud-azure/blob/v2.5.1/README.md

**Comment by niemyjski** [comment]
Yes, it's the latest stable version. The readme should point to what
is stable and in master.

On 4/8/15, David Pilato <notifications@github.com> wrote:
> Are you talking about 2.5.1?
>
> If so, the right documentation lives here:
> https://github.com/elastic/elasticsearch-cloud-azure/blob/v2.5.1/README.md
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/elastic/elasticsearch-cloud-azure/issues/78#issuecomment-91038973


-- 
Thanks
-Blake Niemyjski

**Comment by dadoonet** [comment]
The README has a link to the documentation depending on the version you have.

That say I understand your point.
User expects to read directly the "right" documentation...

Master branch means to you: latest released version
It's not what we did here. Master is a SNAPSHOT version for Elasticsearch master branch (which is also a SNAPSHOT).

I need to think about it. May be we could publish documentation on Elastic.co website and link "latest" to the released version (2.5.1 for now).
It would require more work but would also make this doc searchable from elastic website...

@clintongormley WDYT? Should we create a docs dir as we have for other projects ?

**Comment by dadoonet** [comment]
Closing as the documentation is now properly managed in elasticsearch repo.
